src/components/Image.astro

```diff
@@ -34,6 +34,14 @@ const {
   ...rest
 } = Astro.props
 
+const isLocalAsset = src && (src.startsWith('*/') || src.startsWith('../'))
+let localImage
+
+if (isLocalAsset) {
+  const { default: importedImage } = await import(/* @vite-ignore */ src)
```

Comment:
What does the @vite-ignore do? Is this a hacky workaround for a problem that can be fixed?

Reply:
const importedImage = await import(`../assets/${src}`)